components/observations/submitObservation.ts

```diff
@@ -0,0 +1,115 @@
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import axios from 'axios';
+import * as FileSystem from 'expo-file-system';
+import {isUndefined, omit} from 'lodash';
+import md5 from 'md5';
+
+import {ObservationFormData} from 'components/observations/ObservationFormData';
+import {AvalancheCenterID, MediaItem, mediaItemSchema, Observation} from 'types/nationalAvalancheCenter';
+import {apiDateString} from 'utils/date';
+
+const extensionToMimeType = (extension: string) => {
+  switch (extension.toLowerCase()) {
+    case 'jpeg':
+    case 'jpg':
+      return 'image/jpeg';
+    case 'png':
+      return 'image/png';
+    default:
+      throw new Error(`Unknown mime type for extension: ${extension}`);
```

Comment:
I imagine most folks will be using HEIC, it's standard on both iOS and Android - do we need something special to handle that?

Reply:
don't know yet! need to test on my phone to see if this is transparently handled by the ImagePicker or not.